app/src/app/components/contents/PacketGroup/PacketGroup.tsx

```diff
@@ -1,19 +1,48 @@
-import { useParams } from "react-router-dom";
 import { PacketTable } from "./PacketTable";
+import {useParams} from "react-router-dom";
+import {useGetPacketGroupSummaries} from "../explorer/hooks/useGetPacketGroupSummaries";
+import {HttpStatus} from "../../../../lib/types/HttpStatus";
+import {Unauthorized} from "../common/Unauthorized";
+import {ErrorComponent} from "../common/ErrorComponent";
+import {useGetPacketById} from "../common/hooks/useGetPacketById";
 
 export const PacketGroup = () => {
   const { packetName } = useParams();
+  const { packetGroupSummaries, error: packetGroupError } =
+    useGetPacketGroupSummaries(0, 1, packetName as string);
```

Comment:
Is this just calling the search endpoint? It feels like it would be more efficient to have an endpoint which only gets a single packet group with the known name?

Reply:
There's no guarantee that there is a 'single' packet group with that name (also, the column isn't unique), so we have to do a search.

Reply:
Yes.
The new endpoint would need to also return the id of the latest packet, meaning its quite a lot of extra code just for a neater call here.

Reply:
Too late for the separate ticket I'm afraid. Added a new endpoint that gives back the single packet group's display name, along with the needed id of the latest packet.